<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_Book_Info_Widget</class>
 <widget class="QWidget" name="Change_Book_Info_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="BookNum_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书籍数目</string>
   </property>
  </widget>
  <widget class="QLabel" name="ISBNcode_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ISBN号码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameOfBook">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="BookNum">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="NameOfBook_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConFirm">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IfDelete">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ISBNcode">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="PriceOfBook">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="Writer">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="PriceOfBook_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书籍价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="Writer_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>作者</string>
   </property>
  </widget>
  <widget class="QLabel" name="IfDelete_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>是否删除（勾选）</string>
   </property>
  </widget>
  <widget class="QLabel" name="Publisher_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出版社</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Publisher">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
  </widget>
  <widget class="QLabel" name="Memo_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>萝莉体 第二版</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>书籍简介</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Memo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
